bndtools.core/src/bndtools/views/resolution/ResolutionView.java

```diff
@@ -128,7 +129,11 @@ public class ResolutionView extends ViewPart implements ISelectionListener, IRes
 	private Label				reqsLabel;
 	private Label				capsLabel;
 
+	// hideSelfImportsFilter => show only resolved requirements (strange name)
```

Comment:
Please no such comments. Feel free to change it for a better name but these comments tend to stay for millennia ...

Base idea is to not comment code, only API. If you want to make things more clear, use better names and/or create intermediate variables that show what you want to convey.

This is an old code base, started far before I fully developed all those ideas, and also not with the intention to be a public API, so you will see many violations.

Reply:
> Please no such comments. Feel free to change it for a better name but these comments tend to stay for millennia

Ok sorry, I will create another PR before I forget it. 
While we are on this: Could you explain me the reasoning behind the current name "hideSelfImportsFilter"?
Because I was a bit confused by this, as the tooltip of the button said "Show resolved". 
So I would rename it from  `hideSelfImportsFilter` to `hideUnresolved` (in case we keep the negated naming).
Or would that be wrong?